packages/typegpu/src/createRoot.ts

```diff
@@ -456,7 +456,7 @@ export type CreateRootOptions = {
  */
 export async function createRoot(
   options?: CreateRootOptions,
-): Promise<TgpuRoot> {
+): Promise<ExperimentalTgpuRoot> {
```

Comment:
`createRoot` aka `init` as a public API should return promise of `TgpuRoot`. I assume that's going to be changed in a following PR, oui?